runtime/doc/starting.txt

```diff
@@ -463,13 +458,19 @@ accordingly, proceeding as follows:
 	-  The file ".nvimrc"
 	-  The file ".exrc"
 
-7. Enable syntax highlighting.
+7. Enable filetype detection.
+	This does the same as the command: >
+		:runtime! filetype.lua filetype.vim
```

Comment:
isn't it ".vim first, then .lua"? (it's probably in that order no matter which you put first here, but for the sake of consistency, switching them might be good.)

Reply:
For filetype, it's [Lua then Vim](https://github.com/neovim/neovim/pull/16600#issuecomment-990334535).

Reply:
🤦🏻 

right, we've been over that.